dev-haskell/amazonka-core/amazonka-core-1.6.1.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 # ebuild generated by hackport 0.6.4.9999
 
```

Comment:
This would need a revbump due to http-client bounds change. Sadly I would say to drop this commit, as it's more or less the same as what's there already (unless you'd like to merge in changing the EAPI to 8, which wouldn't hurt for whenever 7 becomes deprecated)

Reply:
I'm pretty sure @trofi told me that we don't need to do a revbump if all we are doing is loosening the bounds of a dependency. I think the logic was that it would not need to be rebuilt for such a small change.

Reply:
Oh, no it was actually the opposite! The dev manual doesn't mention this needing a revision bump, but it actually does... I'll make sure to do this from now on.

Reply:
I'll make sure to do this from now on.